rpcs3/Emu/System.h

```diff
@@ -261,7 +261,7 @@ struct cfg_root : cfg::node
 	{
 		node_core(cfg::node* _this) : cfg::node(_this, "Core") {}
 
-		cfg::_enum<ppu_decoder_type> ppu_decoder{this, "PPU Decoder", ppu_decoder_type::fast};
+		cfg::_enum<ppu_decoder_type> ppu_decoder{this, "PPU Decoder", ppu_decoder_type::llvm};
```

Comment:
Hmm, what do you think about adding an "#ifdef LLVM_AVAILABLE" to enable by default llvm only if you have compiled with it? And keeping "Interpreter (fast)" otherwise as long as LLVM is not mandatory.